ui/src/app/cohort-search/search-bar/search-bar.component.ts

```diff
@@ -68,7 +68,7 @@ export class SearchBarComponent implements OnInit, OnDestroy {
     const optionsSub = this.ngRedux
       .select(autocompleteOptions())
       .subscribe(options => {
-        if (this.searchTerm.length >= 4) {
+        if (this.searchTerm.length >= 2) {
```

Comment:
Should we make this a constant? or maybe create a function that does this check to consolidate this logic since it's in 3 locations?

Reply:
Good idea, done.